eras/shelley/impl/src/Cardano/Ledger/Shelley/Era.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DataKinds    #-}
```

Comment:
I think `cardano-ledger` has different style rules than we do in Consensus. (If I recall correctly, they use `ormolu`.) So tweaks like this are probably out-of-bounds for this PR.